src/types.js

```diff
@@ -0,0 +1,11 @@
+import PropTypes from 'prop-types';
+
+export const TodoListType = PropTypes.shape({
```

Comment:
```suggestion
export const TodoType = PropTypes.shape({
```
это же типы одной todo, а не листа